### ISSUE TITLE
I don't see "agree" in crash reporter privacy dialog

### ISSUE THREAD

**Issue by wb9688** [TeamNewPipe/NewPipe#3657]
I don't see "agree" at all.

_Originally posted by @ale5000-git in https://github.com/TeamNewPipe/NewPipe/issues/3576#issuecomment-634904724_

**Comment by ale5000-git** [comment]
@wb9688: Yes, I'm sure about the missing option; I will post a screenshot this evening.

If you try it on a phone where the screen is "physically" small then 240 isn't so low.

**Comment by wb9688** [comment]
@ale5000-git: It's low as in that my phone and the phone of 2 other people doesn't allow changing it to anywhere under 320 dp.

**Comment by ale5000-git** [comment]
It is because the screen of your phone is bigger.

**Comment by ale5000-git** [comment]
![Screenshot_2020-05-29-22-40-07](https://user-images.githubusercontent.com/15793015/83303980-50f1a400-a1fe-11ea-9541-e41310f51826.png)

**Comment by ShareASmile** [comment]
@ale5000-git do you still have [mentioned android 5.1.1 device](https://github.com/TeamNewPipe/NewPipe/issues/3576#issuecomment-635004806) ?
if yes, can you confirm this small screens device's specific bug on latest version ?

**Comment by ghost** [comment]
Seems like duplicate of #2287
